app/lib/menu.rb

```diff
@@ -0,0 +1,17 @@
+require 'json'
+
+module Menu
+
+  attr_reader :price_list
+
+  def menu_read_price_list
+    file = File.read('./hipstercoffee.json')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.